### ISSUE TITLE
[OTHER] Github handle is not visible in the contributors list

### ISSUE THREAD

**Issue by gitpugal** [narayan954/dummygram#1335]
### What would you like to share?

Hi @narayan954 ,  I have contributed to this repo's db schema last year. but my github handle is still not visible in the contributors list. Can you clarify on that please.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by narayan954** [comment]
> ### What would you like to share?
> Hi @narayan954 , I have contributed to this repo's db schema last year. but my github handle is still not visible in the contributors list. Can you clarify on that please.
> 
> ### Additional information
> _No response_
> 
> ### Record
> * [x]  I agree to follow this project's Code of Conduct
> * [x]  I have checked the existing issues
> * [x]  I'm a GSSoC'23 contributor
> * [x]  I want to work on this issue

I'll check this

**Comment by narayan954** [comment]
> ### What would you like to share?
> Hi @narayan954 , I have contributed to this repo's db schema last year. but my github handle is still not visible in the contributors list. Can you clarify on that please.
> 
> ### Additional information
> _No response_
> 
> ### Record
> * [x]  I agree to follow this project's Code of Conduct
> * [x]  I have checked the existing issues
> * [x]  I'm a GSSoC'23 contributor
> * [x]  I want to work on this issue

Hi @gitpugal sorry for the delay, I checked your contribution and [commit](https://github.com/narayan954/dummygram/commit/c794e75bac939bc82f16eddf8012d86957ece996) to the master branch. First I thought it might've been due to [limit of top 100 contributors](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-a-projects-contributors) but even after getting the whole list, I couldn't find your name and realised perhaps it's because the single commit that got merged into our repo is coauthored by me, hence it's a joint commit, thus github algo might've not considered counting it as your sole contribution. We can have you commit solo and that might add your name but even then we'd need to add multiple commits to rank it under 100.
